apps/site/pages/en/blog/vulnerability/january-2025-security-releases.md

```diff
@@ -1,12 +1,65 @@
 ---
-date: 2025-01-14T03:00:00.000Z
+date: 2025-01-21T03:00:00.000Z
 category: vulnerability
 title: Tuesday, January 21, 2025 Security Releases
 slug: january-2025-security-releases
 layout: blog-post
 author: The Node.js Project
 ---
 
+## Security releases available
+
+Updates are now available for the 23.x, 22.x, 20.x, 18.x Node.js release lines for the
+following issues.
+
+This security release includes the following dependency updates to address public vulnerabilities:
+- undici (v7.2.3, v6.21.1, v5.28.5) on v23.x, v22.x, v20.x, v18.x.
+
+## Worker permission bypass via InternalWorker leak in diagnostics (CVE-2025-23083) - (high)
+
+With the aid of the diagnostics_channel utility, an event can be hooked into whenever a worker thread is created. This is not limited only to workers but also exposes internal workers, where an instance of them can be fetched, and its constructor can be grabbed and reinstated for malicious usage.
+
+This vulnerability affects Permission Model users (--permission) on Node.js v20, v22, and v23.
+
+Impact:
+
+- This vulnerability affects all users in active release lines: 20.x, 22.x, 23.x
+
+Thank you, to leodog896 for reporting this vulnerability and thank you RafaelGSS for fixing it.
+
+## Path traversal by drive name in Windows environment (CVE-2025-23084) - (medium)
+
+A vulnerability has been identified in Node.js, specifically affecting the handling of drive names in the Windows environment. Certain Node.js functions do not treat drive names as special on Windows. As a result, although Node.js assumes a relative path, it actually refers to the root directory.
+
+On Windows, a path that does not start with the file separator is treated as relative to the current directory.
+
+This vulnerability affects Windows users of `path.join` API.
```

Comment:
Should this sentence include the impacted release lines like the other sections do?

Reply:
All release lines were impacted, but I think it's ok to mention

Reply:
Wait, we are mentioning it already, in the line below.

Reply:
Yeh, the line below covers it, but for the other vulns we repeat it in this final sentence too. Was just a nit for consistency, not too worried either way.